roles/edpm_ovn/defaults/main.yml

```diff
@@ -78,16 +78,19 @@ edpm_ovn_ovs_external_ids:
   ovn-bridge-mappings: "{{ edpm_ovn_bridge_mappings | join(',') }}"
   ovn-chassis-mac-mappings:
     "{%- set chassis_mac_mappings = [] -%}
-     {%- for physnet, mac_prefix in edpm_ovn_chassis_mac_mapping_prefixes.items() -%}
-       {{ chassis_mac_mappings.append([physnet, mac_prefix | community.general.random_mac(seed=edpm_ovn_chassis_mac_mapping_seed)] | join(':')) }}
-     {%- endfor -%}
-     {{ chassis_mac_mappings | join(',') }}"
+    {%- for physnet, mac_prefix in edpm_ovn_chassis_mac_mapping_prefixes.items() -%}
+    {{ chassis_mac_mappings.append([physnet, mac_prefix | community.general.random_mac(seed=edpm_ovn_chassis_mac_mapping_seed)] |
+      join(':')) }}{%- endfor -%}
+    {{ chassis_mac_mappings | join(',') }}"
   ovn-encap-ip: "{{ edpm_ovn_encap_ip }}"
   ovn-encap-type: "{{ edpm_ovn_encap_type }}"
   ovn-match-northd-version: true
   ovn-monitor-all: true
   ovn-openflow-probe-interval: "{{ edpm_ovn_of_probe_interval }}"
-  ovn-remote: "{% set db_addresses = [] %}{% for host in edpm_ovn_dbs %}{{ db_addresses.append([edpm_ovn_protocol, host, edpm_ovn_sb_server_port] | join(':')) }}{% endfor %}{{ db_addresses | join(',') }}"
+  ovn-remote:
+    "{% set db_addresses = [] %}{% for host in edpm_ovn_dbs %}
```

Comment:
failures in molecule are relavant, need to fix extra space in ovn-remote

Reply:
indeed, thx @karelyatin !